### ISSUE TITLE
v0.0.10 does not run on macOS arm64

### ISSUE THREAD

**Issue by bits01** [marcopaganini/termotp#8]
It just gets killed on M2 arm64 macOS 14.7.1 (Sonoma):

```
> file ./termotp
./termotp: Mach-O 64-bit executable arm64

> ./termotp -version
[1]    47721 killed     ./termotp -version
```

v0.0.9 runs fine.

**Comment by marcopaganini** [comment]
This is probably caused by the upx binary compressor. I've removed it in 45175e7 and released v0.0.11 with a fix. Can you confirm it's fixed?

**Comment by bits01** [comment]
It works! Thanks for the quick fix.
